src/DynamoCoreWpf/UI/Themes/Modern/DynamoModern.xaml

```diff
@@ -517,7 +517,99 @@
         </Style.Triggers>
     </Style>
 
+    <Style x:Key="TextButtonStyle" TargetType="Button">
+        <Setter Property="HorizontalAlignment" Value="Right" />
+        <Setter Property="Cursor" Value="Hand" />
+        <Setter Property="VerticalAlignment" Value="Top" />
+        <Setter Property="Margin" Value="18,0,0,0" />
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="Button">
+                    <TextBlock x:Name="textBlock"
+                               HorizontalAlignment="Center"
+                               VerticalAlignment="Center"
+                               FontFamily="{StaticResource ArtifaktElementBold}"
+                               FontSize="14px"
+                               Foreground="#6DD2FF"
```

Comment:
This color has been defined a couple other places, can we define it in `Modern\DynamoColorsAndBrushes.xaml`?

Reply:
#6DD2FF is extremely close to Blue300Brush (#6AC0E7) so I'll substitute that assuming that was the intention :)